### PR TITLE
Prepare Event Behaviour for P&S

### PR DESCRIPTION
**Pull request by ehsanmx**
* Fix getEventTransfer* issues
* Fix the bugs which brakes install script

### CODE REVIEW

File: src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php [hunk 1]
```diff
@@ -42,6 +39,11 @@ class TriggerManager implements TriggerManagerInterface
      */
     protected $config;
 
+    /**
+     * @var bool
+     */
+    protected static $eventBehaviorTableExists;
```
Comment (dereuromark) [inline src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php]: should this default to bool?

**Comment by dereuromark** [comment]
Apart from my comment :+1:

**Comment by ehsanmx** [comment]
This should be release as a patch **new version is 0.1.2**

**Comment by ehsanmx** [comment]
# Release note:
This Release will fix the issue which breaks the install script when the Publish and Synchronization feature is enabled.

## Change logs:
* The EventEntityTransferFilter.php::getEventTransferForeignKeys() method has fixed by checking null index columns and filtering them.
* Add a PreCheck for TriggerManager to not fail when Database is not ready yet.

**Pull request merged**